dev-requirements.txt

```diff
@@ -14,17 +14,4 @@ python-magic==0.4.18
 messytables==0.15.2
 progressbar==2.5
 
--e git+https://github.com/ckan/ckanext-dcat.git@v1.2.0#egg=ckanext-dcat
--e git+https://github.com/ckan/ckanext-scheming.git@release-2.1.0#egg=ckanext-scheming
--e git+https://github.com/qld-gov-au/ckanext-archiver.git@2.1.1-qgov.12#egg=ckanext-archiver
--e git+https://github.com/qld-gov-au/ckanext-csrf-filter.git@1.1.5#egg=ckanext-csrf-filter
--e git+https://github.com/qld-gov-au/ckanext-datarequests.git@2.2.1-qgov.3#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-odi-certificates.git@1.0.1#egg=ckanext-odi-certificates
--e git+https://github.com/qld-gov-au/ckanext-qa.git@2.0.3-qgov.6#egg=ckanext-qa
--e git+https://github.com/qld-gov-au/ckanext-qgov.git@5.0.3#egg=ckanext-qgov
--e git+https://github.com/qld-gov-au/ckanext-report.git@0.3#egg=ckanext-report
--e git+https://github.com/qld-gov-au/ckanext-resource-type-validation.git@1.0.3#egg=ckanext-resource-type-validation
--e git+https://github.com/qld-gov-au/ckanext-ssm-config.git@0.0.2#egg=ckanext-ssm-config
--e git+https://github.com/qld-gov-au/ckanext-validation.git@v0.0.8-qgov.5#egg=ckanext-validation
--e git+https://github.com/qld-gov-au/ckanext-xloader.git@0.10.0-qgov.1#egg=ckanext-xloader
--e git+https://github.com/qld-gov-au/ckanext-ytp-comments.git@2.5.0-qgov.7#egg=ckanext-ytp-comments
+pika>=1.1.0,<1.3.0 # resolve harvester conflict
```

Comment:
Why move away from pip dev-requirements.txt

We have standardised on pip installs.

I can't accept this unless you provide a public backing issue ticket on why another process should be utilised.

Reply:
Will be fixed and pushed by Alex shortly.